tests/maven.Tests.ps1

```diff
@@ -47,6 +47,12 @@ Describe "$SUT_TAG create test container" {
     $exitCode | Should -Be 0
     $stdout | Should -Match "Apache Maven $version"
   }
+  It 'runs as non admin user' {
+    $version = $(Get-Content -Path "$PSScriptRoot/../$SUT_TAG/Dockerfile" | Select-String -Pattern 'ARG MAVEN_VERSION.*' | ForEach-Object { $_ -replace 'ARG MAVEN_VERSION=','' })
+    $exitCode, $stdout, $stderr = Run-Program -Cmd "docker.exe" -Params "run --rm ${SUT_IMAGE}:${SUT_TAG} echo $env:USERNAME"
```

Comment:
```suggestion
    $exitCode, $stdout, $stderr = Run-Program -Cmd "docker.exe" -Params "run --rm ${SUT_IMAGE}:${SUT_TAG} echo `$env:USERNAME"
```

The escape character in PowerShell is the backtick, I forgot that. This passes locally when I run it with this. Where do I update the surefire version?